resources/monitoring/README.md

```diff
@@ -8,6 +8,24 @@ This chart installs the Prometheus operator along with [Grafana](https://grafana
 
 All the monitoring components run in the `kyma-system` Namespace.
 
+## Custom Resource Definitions
+
+Custom resource definitions for `crd-alertmanager.yaml`, `crd-podmonitor.yaml`, and `crd-servicemonitor.yaml` have been moved to the `resources/cluster-essentials/templates` folder and will be maintained there. Along with the next update of the Prometheus Operator, update the listed files in the [Cluster Essentials](https://github.com/kyma-project/kyma/tree/master/resources/cluster-essentials) chart.
+
+## Kyma Customizations
+
+All the extra files added to this chart for the usage in Kyma are collected under `kyma-additions` folders in this chart or the sub-charts.
+
+All the in-line customizations are commented on in the respective files. Search for `Customization` to see them all.
+
+## Kyma Dashboards
+
+Kyma comes with some extra dashboards to enable monitoring its components. All the Kyma dashboards can be found under [kyma-dashboards](templates/grafana/kyma-dashboards). For more information on the dashboards, see [this document](charts/grafana/README.md).
+
+## Kyma Alert Rules
+
+Kyma comes with some extra alert rules to enable alerting for its components. All the Kyma alert rules can be found under [kyma-rules](templates/prometheus/kyma-rules).
+
 ## Details
 
-* For details on Grafana implementation in Kyma, see [this document](charts/grafana/README.md).
+* For details on Grafana usage in Kyma, see [this document](charts/grafana/README.md).
```

Comment:
```suggestion
For details on Grafana usage in Kyma, see [this document](charts/grafana/README.md).
```
I would remove the bullet point because it's just one sentence.